src/composables/profile.js

```diff
@@ -81,6 +81,7 @@ export const useProfile = () => {
         validate(key);
       }
       if (!isErrorPresent(errorMessages)) {
+        const usersRef = collection(db, "users");
         const q = query(usersRef, where("uid", "==", userDetails.value?.uid));
```

Comment:
I think this is the wrong way to get a user
why can't it be done like this

const db = firebase.firestore();

const userDoc = await db.collection('users').doc(userID).get(); @aayushbatra02